index.html

```diff
@@ -63,6 +63,224 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+            </div>
+        </div>
+
+        </div>
+        <!-- Work section item three -->
+        <div class="works-grid-item">
+            <div class="template-img flex-item-width">
+                <img src="img/Snapshoot Portfolio5.png" alt="">
+            </div>
+            <div class="Tonic flex-item-width">
+                <h2>Tonic</h2>
+            </div>
+            <div class="ol-list flex-item-width">
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ol>
+            </div>
+            <div class="flex-item-width paragraph">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="flex-item-width buttons-flix">
+                <ul class="ul-list">
+                    <button class="buttons"> <li>HTML</li></button>
+                    <button class="buttons"><li>CSS</li></button>
+                    <button class="buttons">
+                <li>JavaScript</li></button>
+                </ul>
+            </div>
+            <br>
+            <div class="flex-item-width">
+                <button id="seeProject-button">See Project</button>
+            </div>
+        </div>
+        <!-- Work section item four -->
+        <div class="works-grid-item">
+            <div class="template-img flex-item-width">
+                <img src="img/Snapshoot Portfolio 2.png" alt="">
+            </div>
+            <div class="Tonic flex-item-width">
+                <h2>Multi-Post Stories
+                </h2>
+            </div>
+            <div class="ol-list flex-item-width">
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ol>
+            </div>
+            <div class="flex-item-width paragraph">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="flex-item-width buttons-flix">
+                <ul class="ul-list">
+                    <button class="buttons"> <li>HTML</li></button>
+                    <button class="buttons"><li>CSS</li></button>
+                    <button class="buttons">
+            <li>JavaScript</li></button>
+                </ul>
+            </div>
+            <br>
+            <div class="flex-item-width">
+                <button id="seeProject-button">See Project</button>
+            </div>
+        </div>
+        <br><br>
+    </section>
+
+    <!-- About Myself Section -->
+    <section class="myself-container">
+
+        <div class="myself-flex-items flex-item-width heading2">
+            <h2>About Myself</h2>
+        </div>
```

Comment:
- [ ] Just to re-state what the previous reviewer suggested, kindly check the compared designs, and see the changes needed 👍 

## Your work:
![image](https://user-images.githubusercontent.com/50990925/235326021-352e6047-0b3b-495b-91c2-42ba2ef086e5.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/235326035-c3c69441-3a2e-476e-9c93-edd6e04997f1.png)